packages/opentelemetry-tracer-web/src/StackScopeManager.ts

```diff
@@ -0,0 +1,123 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ScopeManager } from '@opentelemetry/scope-base';
+
+/**
+ * Stack Scope Manager for managing the state in web
+ * it doesn't fully support the async calls though
+ */
+export class StackScopeManager implements ScopeManager {
+  /**
+   * whether the scope manager is enabled or not
+   */
+  private _enabled = false;
+
+  /**
+   * Keeps the reference to current scope
+   */
+  public _currentScope: any;
```

Comment:
Since we aren't actually doing anything with the scope, can we use `unknown` here?